apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -0,0 +1,55 @@
+var Visualization = require('./Visualization');
+var GameButtons = require('./GameButtons');
+
+var styles = {
+  hidden: {
+    display: 'none'
+  }
+};
+
+/**
+ * Equivalent of visualizationColumn.html.ejs. Initially only supporting
+ * portions used by App Lab
+ */
+var VisualizationColumn = React.createClass({
+  propTypes: {
+    isProjectLevel: React.PropTypes.bool.isRequired,
+    isSubmittable: React.PropTypes.bool.isRequired,
+    isSubmitted: React.PropTypes.bool.isRequired,
+  },
+
+  render: function () {
+    return (
+      <div>
+        <Visualization/>
+        <GameButtons
+            isProjectLevel={this.props.isProjectLevel}
+            isSubmittable={this.props.isSubmittable}
+            isSubmitted={this.props.isSubmitted}
```

Comment:
Especially after seeing that `VisualizationColumn` only passes these props through to `GameButtons` - leave VisualizationColumn unconnected and connect GameButtons.  Less is more.